doc/source/ray-air/examples/BUILD

```diff
@@ -46,6 +46,8 @@ py_test_run_all_notebooks(
         "feast_example.ipynb",  # REGRESSION
         "upload_to_comet_ml.ipynb",  # Needs credentials
         "upload_to_wandb.ipynb",  # Needs credentials
+        "gptj_batch_prediction.ipynb",  # Requires GPUs
+        "stablediffusion_batch_prediction.ipynb",  # Requires GPUs
```

Comment:
shouldn't this go into the GPU build?

Reply:
this is the exclude list for the CPU build :)